assignment/Assignment2.kt

```diff
@@ -0,0 +1,50 @@
+package assignment
+
+import java.util.Scanner
+
+class KStack<T>{
+    private val stack = mutableListOf<T>()
+
+    fun push(element: T){
+        stack.add(element)
+    }
+    fun pop(): T?{
+        if(stack.size == 0)
+            return null
+        return stack.removeLast()
+    }
```

Comment:
```suggestion
    fun pop(): T? = if (stack.isEmpty()) null else stack.removeLast()
```

- size가 0이다라는 구문 대신 isEmpty를 활용할 수 있습니다
- 리턴하실때 조건문을 활용하실 수 있습니다!